packages/markdown/loader.js

```diff
@@ -419,6 +420,27 @@ module.exports = async function demoLoader() {
         // TS version of the demo doesn't exist. This is fine.
       }
 
+      /**
+       * Inserts the moduleData into the relativeModules object
+       * @param {*} moduleData
+       * @param {*} variant
+       * @example updateRelativeModules({module: 'constants.js', raw: ... }, 'JS') => demos[demoName].relativeModules[variant].push(moduleData)
+       */
+      function updateRelativeModules(moduleData, variant) {
```

Comment:
We could perhaps move this function definition outside of the loop by adding `demoName` as the first parameter